sdl-lang/src/main/resources/atlasExporter.purview.conf

```diff
@@ -0,0 +1,40 @@
+
+atlasServer {
+  url: "https://xyz.purview.azure.com/catalog/"
+  auth {
+    token: "CLEAR#xyz"
```

Comment:
If we want to use Purview, we should implement the proper OAuth for authentication. Using a token is just for testing purposes as you need to get the token manually first.

Reply:
Ok, could be a topic in issue #426.